Dockerfile

```diff
@@ -1,17 +1,19 @@
 # Build the manager binary
-FROM golang:1.11.4-alpine3.8 as builder
+FROM golang:1.12.10-alpine3.10 as builder
+
+ENV SRC_DIR=/go/src/github.com/kyma-incubator/octopus
 
 # Copy in the go src
-WORKDIR /go/src/github.com/kyma-incubator/octopus
-COPY pkg/    pkg/
-COPY cmd/    cmd/
-COPY vendor/ vendor/
+WORKDIR $SRC_DIR
+ADD . $SRC_DIR
```

Comment:
if you want to change the way of copying files to copy everything then add also proper .dockerignore file

Reply:
I will revert that change and copy only necessary files